### DISCUSSION TITLE
Status of the repo for Simple Lit

### DISCUSSION THREAD

**Discussion by Zallist** [Zallist/unity-universal-shadergraph-extensions#10] [Announcements]
Looks to be all completed to the exact same standards as Unity itself.
No new features will be added to this now, unless they're also added to the Lit/Unlit master nodes. Only bug fixes.

If URP version 14/15/16 breaks this and an issue hasn't been raised already, raise one. Hopefully Unity will adopt this properly before they have a chance to break it, though.

**Comment by SozapJacob** [comment]
I tried this in unity 6 and can't get it to work. Unity have not updated any news regarding to their own implementation. Any chance that this repo will be updated to support Unity 6?

**Reply by Zallist** [reply]
I have no intention of updating this repository at for unity 6+.
Relevant files if you want to take a crack at updating it:

* https://github.com/Unity-Technologies/Graphics/blob/master/Packages/com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs
* https://github.com/Unity-Technologies/Graphics/blob/master/Packages/com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/PBRForwardPass.hlsl
* https://github.com/Unity-Technologies/Graphics/blob/master/Packages/com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/PBRGBufferPass.hlsl

Recommended approach:
* Remove all the `#if` checks and reduce the code down to the absolute minimum that worked on 2022.2+
* Use a comparison tool and compare each of the 3 relevant files to the latest versions in unity's repo
* Copy over any changes that look like they might be broken.

There's probably only a handful of actual changes that are needed to each file to make them work, since it doesn't seem like they outright broke the integration.

**Reply by SozapJacob** [reply]
Alright, thanks!
